ion/type.go

```diff
@@ -90,6 +90,26 @@ func (t Type) String() string {
 	}
 }
 
+// IsScalar determines if the type is a scalar type
+func (t Type) IsScalar() bool {
+	switch t {
+	case NullType, BoolType, IntType, FloatType, DecimalType, TimestampType, SymbolType, StringType, ClobType, BlobType:
```

Comment:
Instead of the switch statement, we can just check the type against the int value of the enum

Reply:
Done.